flink-dist/src/main/flink-bin/conf/log4j-cli.properties

```diff
@@ -19,10 +19,11 @@
 log4j.rootLogger=INFO, file
 
 # Log all infos in the given file
-log4j.appender.file=org.apache.log4j.FileAppender
+log4j.appender.file=org.apache.log4j.DailyRollingFileAppender
 log4j.appender.file.file=${log.file}
 log4j.appender.file.append=false
 log4j.appender.file.layout=org.apache.log4j.PatternLayout
+log4j.appender.file.DatePattern=.yyyy-MM-dd
```

Comment:
add `log4j.appender.file.MaxFileSize` and  `log4j.appender.file.MaxBackupIndex` to limit the total size of log files.

Reply:
Will add soon.